tests/integration/components/public/event-map-test.js

```diff
@@ -9,5 +9,5 @@ let event = Object.create({ latitude: 37.7833, longitude: -122.4167, locationNam
 test('it renders', function(assert) {
   this.set('event', event);
   this.render(hbs `{{public/event-map event=event}}`);
-  assert.equal(this.$('.address p').text(), 'Sample event location address');
+  assert.ok(this.$().html().trim().includes('Sample event location address'));
```

Comment:
No @CosmicCoder96 . Ideally this is how we want all tests to be. We want to actually check if the data is correctly displayed. Please revert this back to how it was.